public-ui/src/app/data-browser/views/survey-view/survey-view.component.html

```diff
@@ -1,346 +1,398 @@
-  <div class="survey-view">
-    <div class="page-header">
-      <div class="title-container" *ngIf="surveyConceptId && surveyName && surveyDescription">
-        <h1> {{surveyName}} </h1>
-        <div class="desc-container">
-          <p class="body-default help-text survey-desc">{{surveyDescription}} Survey questions appear in the order in which participants took the survey.</p>
-        </div>
-      </div>
-      <div class="survey-note"> <span class="bold-note-heading">Note</span>: The data on this page are:
-        <ul class="survey-note">
-          <li>Gathered directly from participants through electronic surveys</li>
-          <li>Grouped into bins of 20 to protect privacy</li>
-        </ul>
-        For more information about this survey, please visit the <a target="_blank" href="https://www.researchallofus.org/data-sources/survey-explorer" rel="noopener noreferrer">Survey Explorer</a>
+<div class="survey-view">
+  <div class="page-header">
+    <div class="title-container" *ngIf="surveyConceptId && surveyName && surveyDescription">
+      <h1> {{surveyName}} </h1>
+      <div class="desc-container">
+        <p class="body-default help-text survey-desc">{{surveyDescription}} Survey questions appear in the order in
+          which participants took the survey.</p>
       </div>
     </div>
-    <div class="search-bar-container">
-      <div id="db-search-bar">
-        <clr-icon shape="search" class="is-info search-icon"></clr-icon>
-        <input type="text" aria-label="Survey Search" id="search-db" placeholder="Keyword Search" [formControl]="searchText" style="height:auto" autofocus>
-        <div *ngIf="searchText.value"  class="clear-icon" (click)="clearSearch()"><i class="far fa-times fa-1x clear-search-icon"></i></div>
-      </div>
+    <button *ngIf="copeFlag && domainId == 'covid-19-participant-experience'" (click)="showStatement = true" class="disclaimer-btn">IMPORTANT CONSIDERATIONS FOR COPE SURVEY - LEARN MORE</button>
```

Comment:
Filtering on survey concept id is better may be. Incase domain name changes later.